DEVELOPMENT.md

```diff
@@ -79,5 +79,36 @@ Look at the logs in the Log Group `/aws/sagemaker/studio` and the Log Stream `$d
 
 Ensure your SageMaker Studio execution role has permissions to pull the image from ECR. See the [ECR documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/security_iam_id-based-policy-examples.html) for IAM permissions required to pull an image.
 
+### Files With Unsupported UIDs/GIDs
 
+SageMaker only supports UIDs/GIDs less than 65535. You should ensure that your container does not have any layers containing a file with a GID/UID higher than 65535.
```

Comment:
Let's link to the documentation that explains this limit.

Reply:
done